pipeline/src/test_util.rs

```diff
@@ -28,20 +31,15 @@ pub fn verify_asm_string<T: FieldElement>(
     inputs: Vec<T>,
     external_witness_values: Vec<(String, Vec<T>)>,
 ) {
-    let pipeline =
-        Pipeline::default().from_asm_string(contents.to_string(), Some(PathBuf::from(file_name)));
-    verify_pipeline(pipeline, inputs, external_witness_values)
+    let pipeline = Pipeline::default()
+        .from_asm_string(contents.to_string(), Some(PathBuf::from(file_name)))
+        .with_prover_inputs(inputs)
+        .add_external_witness_values(external_witness_values);
+    verify_pipeline(pipeline)
 }
 
-pub fn verify_pipeline<T: FieldElement>(
-    pipeline: Pipeline<T>,
-    inputs: Vec<T>,
-    external_witness_values: Vec<(String, Vec<T>)>,
```

Comment:
Not really related to this PR, but I thought having these (optional) arguments here isn't really in the spirit of the pipeline; the caller can just add them if needed.